platform-core-module/src/main/java/com/softicar/platform/core/module/program/execution/cleanup/ProgramExecutionsCleanupProgram.java

```diff
@@ -27,7 +27,6 @@ public class ProgramExecutionsCleanupProgram implements IProgram {
 	@Override
 	public void executeProgram() {
 
-		new ProgramExecutionsCleaner().cleanupOrphanedRecords();
```

Comment:
- second paragraph in Javadoc must be removed
- class should be renamed to `ProgramExecutionsDeleterProgram`, maybe rename both classes:
  - `OldProgramExecutionsDeleterProgram`
  - `OldProgramExecutionsDeleter`

Reply:
ok